### DISCUSSION TITLE
Messenger not working in production

### DISCUSSION THREAD

**Discussion by minato223** [symfony/symfony#53614] [Q&A]
Okay, I have just deployed my application on my server and in my .env file, when APP_ENV is "dev", my email service with messenger works fine but once it changes to "prod", it no longer works ,

I did a test without messenger and it works

![Capture d’écran du 2024-01-23 15-31-23](https://github.com/symfony/symfony/assets/71100950/7ecf41da-7431-401f-80df-518a98f9e66d)

**Comment by ghost** [comment]
I'm sorry but I don't understand the issue description. If you can reword it then maybe I can help.

**Reply by minato223** [reply]
Okay, I have just deployed my application on my server and in my .env file, when APP_ENV is "dev", my email service with messenger works fine but once it changes to "prod", it no longer works ,

I did a test without messenger and it works

**Reply by ghost** [reply]
what does it mean it doesn't work? what happens? any logs?

**Reply by minato223** [reply]
strangely I don't have a production log, I don't know why
I changed the permissions of the var folder but despite all that I have no log
![Capture d’écran du 2024-01-29 09-51-34](https://github.com/symfony/symfony/assets/71100950/6cf7f82b-d8b5-4cc0-89cf-cc12b1abd044)
